appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockFusionTablesControl.java

```diff
@@ -27,7 +30,8 @@
 public class MockFusionTablesControl extends MockNonVisibleComponent {
 
   public static final String TYPE = "FusiontablesControl";
-  private static boolean warningGiven = false; // Whether or not we have given experimental warning
+  private static boolean warningGiven = false; // Whether or not we have given experimental warning (on drag out)
```

Comment:
We should set liveWarningGiven to true if we also set warningGiven to true. The reason for this is if you create a blank project and drag in a new FusiontablesControl you will end up with two dialogs, one for the drag out and one for the attach. If you've received the drag out dialog it doesn't make sense to then immediately get the onAttach dialog. Of course, it might be the other way around...

Reply:
To play devil's advocate, if you got the popup upon opening the project and STILL try to add a new control, maybe you need to be reminded again....